scripts/blackjack.coffee

```diff
@@ -0,0 +1,263 @@
+# Description:
+#   Play a game of blackjack against the bot
+#
+# Commands:
+#   hubot hitme - Instruct the bot to deal you another card
+#   hubot stay - Inform the bot that you are satisfied with your hand.
+#   hubot bjstatus - Your hand and the dealer's hand
```

Comment:
Willing to change to something like 'cards', but given the audience of pushbot I imagine this variant to have some success.